copt/homotopy.py

```diff
@@ -0,0 +1,102 @@
+"""Homotopy Methods."""
+import json
+import os
+import sys
+import warnings
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import io as sio
+from scipy import linalg, optimize
+
+import copt
+from copt import datasets, utils
+
+EPS = np.finfo(np.float32).eps
+
+# TODO :ref:`hcgm`
```

Comment:
Please remove TODO or at least make it clearer

Reply:
Whoops, sorry about that. Removed.